circle/functions

```diff
@@ -429,13 +429,14 @@ chart_update_version() {
 }
 
 get_image_build_dir() {
-    local RELEASE_SERIE=${1}
+    local RELEASE_BRANCH=${1}
     local BASE_IMAGE=${2}
+    local DOCKERFILE=${3:-Dockerfile}
```

Comment:
This is not required . There is already a macro named `DOCKERFILE` defined in `functions`.